imageroot/update-module.d/05database_upgrade

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2025 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+# redirect the standard output  to the standard error
+exec 1>&2
+
+# test if the postgres-app is active
+if ! systemctl is-active -q --user postgres.service; then
```

Comment:
What happens if, before the update, the database service was stopped or crashed?
The container will start with the new postgres version but with the old data format.